crates/iota/src/name_commands.rs

```diff
@@ -2132,7 +2132,10 @@ async fn select_coin_arg_for_payment(
                 .await?;
             let mut balance = 0;
             if let IotaClientCommandResult::Gas(coins) = gas_result {
-                if coins.len() == 1 {
+                // Ignore coins < 0.009 IOTA (9_000_000 NANOs), because they are probably not
+                // sufficient for the gas payment
+                let eligible_coins = coins.iter().filter(|c| c.value() >= 9_000_000);
```

Comment:
Can we create a constant for that and move the documentation there? I suggest: `MIN_COIN_AMOUNT_FOR_GAS_PAYMENT` or smth?